pom.xml

```
@@ -379,6 +379,16 @@ under the License.
 		                                                                        <projectName>Apache Flink</projectName>
 		                                                                </transformer>
 									</transformers>
+									<filters>
+										<filter>
+											<artifact>*:*</artifact>
```

```suggestion
											<artifact>*</artifact>
```
nit: this works as well ;)